spec/models/document_spec.rb

```diff
@@ -208,6 +208,8 @@
           value: 'other-value-1'
         )
       end
+    end
+  end
```

Comment:
What happened here? Can you fix it up?

Reply:
Not sure why it wasn't there at the beginning but it is needed to end the tests